formats/policy/frontend/examples/policy_area.json

```diff
@@ -29,6 +29,28 @@
           "key": "government_name",
           "display_as_result_metadata": true,
           "filterable": false
+        },
```

Comment:
Did you mean to include these changes in this PR?  It doesn't look related to the rest of the PR.